cookbooks/aws-parallelcluster-common/resources/efs/partial/_mount_umount.rb

```diff
@@ -18,8 +18,6 @@
 property :efs_encryption_in_transit_array, Array, required: false
 property :efs_iam_authorization_array, Array, required: false
 
-default_action :mount
```

Comment:
Why did you remove the default action?
are we sure we're calling it always with an action specified?

Reply:
Yes, the default action is `:install_utils`, this was actually a bug and it made `efs:install_utils` test fail on EC2.

Reply:
so I'd add:
```
default_action :install_utils
```
what do you think? This is aligned with all the other resources on which we're always defining a default action.